pkg/pillar/cmd/msrv/msrv_test.go

```diff
@@ -205,6 +205,26 @@ func TestRequestPatchEnvelopes(t *testing.T) {
 						URL:      "a.txt",
 					},
 				},
+				CipherBlobs: []types.BinaryCipherBlob{
```

Comment:
@naiming-zededa I know I'm a bit late to the party, but will there be any encrypted tests?

Reply:
@uncleDecart I don't currently plan to have one, we can think about how it will work later.